### DISCUSSION TITLE
Sequential Slides in Slide_Player:

### DISCUSSION THREAD

**Discussion by Steel1790** [missionpinball/missionpinball#82] [Q&A] [answered]
Hello, 
currently working on trying to get it to go from the razorwirevideo slide to Spinner1 slide when razorwirevideo slide is done. I tried in slide_player by saying when slide_razorwirevideo_completed: Spinner1 , but nothing. Here's the code below. Open to ideas as I feel like I've exhausted quite a few options and can't get the 2nd slide to start after the first one.

#config_version=5                 
mode:
  start_events: mode_cellar_complete
  stop_events: stop_mode_razorwire, ball_ended, stop_mode_cellar
  
slides:

  razorwirevideo:
    widgets:
    - type: video
      video: razorwirevideo  
      expire: 6.7s
      
      
  Spinner1:
    widgets:
    - type: Text
      text: Shoot The Spinners!
      font_name: TrajanProRegular
      font_size: 75
      color: 65fe08
    - type: Text
      text: (player|your_timer_variable_times_100)
      font_size: 50
      y: 85%
    - type: Text
      text: You Have 60 Seconds To Hit 30 Spins
      color: yellow
      font_name: TrajanProRegular
      font_size: 35
      y: 90
      animations:
       pre_show_slide:
        - property: opacity
          value: 0
          duration: 1s
        - property: opacity
          value: 1
          duration: 1s
          repeat: true
    - type: image
      image: razorwireimg
      opacity: 0.65
    transition:
     type: fade
     duration: 0.2s

timers:
  your_timer:
    start_value: 0
    end_value: 60
    control_events:
      - action: start
        event: mode_razorwire_started
  

shot_groups:
  spinner:
    shots: spinner

variable_player:
  spinner_hit:
    score: 2000000
    
  timer_your_timer_tick:
    your_timer_variable_times_100:
      int: device.timers.your_timer.ticks_remaining
      action: set
  
shots:
  spinner:
    switch: s_spinner
      
slide_player:
  mode_razorwire_started: razorwirevideo
  slide_razorwirevideo_completed: Spinner1

**Comment by atummons** [comment, marked as answer]
The first question is do you ever see that event in your log/mpf monitor? If not, then the following stuff keying on that won't happen.

In looking at the config docs, I don't see a slide_xx_complete, but there is a slide_xx_removed. So I would try swapping that from your slide player and see if that works.

**Reply by Steel1790** [reply]
I’ll try that shortly and report back. Would the expire: 6.7s qualify it for being considered removed?

**Reply by Steel1790** [reply]
That works. Only slight issue is there’s like a 0.5 second delay where it shows the first slide. Then a gap of the default screen. Then 0.5 seconds later launches the second.
Any suggestions to make that smoother ?

**Reply by atummons** [reply]
Glad to hear that helped! There are many options to solve this. One option is to make them both play on the same event and set the priority of the video higher. This they're both there the whole time. Then when the video expires (removes itself) the lower slid is already there. Then no awkward transition.

Other options get more complex to have other events/timers that handle the other stuff. But I'd do lower priority first.
